src/DBFTPlugin/ConsensusMessage.cs

```diff
@@ -6,6 +6,7 @@ namespace Neo.Consensus
 {
     public abstract class ConsensusMessage : ISerializable
     {
+        protected readonly byte validatorsCount;
```

Comment:
Why do we need this?

Reply:
For index check in deserialize.
I removed `DBFTPlugin.System`.

Reply:
Maybe we can check it in `Verify()`?

Reply:
Will do. It'll looks better.

Reply:
Changed except `RecoveryMessage`, please take a look